dotnetv3/cross-service/PhotoAssetManager/README.md

```diff
@@ -0,0 +1,106 @@
+#  Create a photo asset management application with the AWS SDK for .NET (v3)
+
+## Overview
+
+This example shows you how to use the AWS SDK for .NET (v3) to create a photo management application using AWS services and a serverless architecture.
+
+The Photo Asset Management (PAM) example app uses Amazon Rekognition to categorize images, which are stored with Amazon S3 Intelligent-Tiering for cost savings. Users can upload new images that are analyzed with label detection. Those labels are then stored in an Amazon DynamoDB table. Users can later request a bundle of images associated with a list of labels. When images are requested, they are retrieved from Amazon S3, zipped, and the user is sent a link to download the resulting zip file.
+For more details on the complete application, see the [PAM application directory](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/applications/photo-asset-manager) in this repository.
+
+![pam_overview.png](pam_overview.png)
+
+### Frontend
+The frontend is a single React app that uses the Cloudscape Design System. With the React app, users can authenticate with an Amazon Cognito flow. The app is deployed to an Amazon S3 bucket using the provided AWS CDK stacks and publicly exposed using an Amazon CloudFront distribution. The name is dynamically created during the AWS CDK deployment.
```

Comment:
Revise third sentence: 
The app is deployed to an Amazon Simple Storage Service (Amazon S3) bucket using the provided AWS Cloud Development Kit (AWS CDK) stacks. It is publicly exposed by using an Amazon CloudFront distribution.